src/Admin/AbstractAdmin.php

```diff
@@ -72,6 +72,34 @@ abstract class AbstractAdmin extends AbstractTaggedAdmin implements AdminInterfa
             Doctrine\\\Orm|Doctrine\\\Phpcr|Doctrine\\\MongoDB|Doctrine\\\CouchDB
         )\\\(.*)@x';
 
+    private const ACTION_TREE = 1;
+    // private const ACTION_CREATE = 2; // not used now
+    private const ACTION_SHOW = 4;
+    private const ACTION_EDIT = 8;
+    private const ACTION_DELETE = 16;
+    private const ACTION_ACL = 32;
+    private const ACTION_HISTORY = 64;
+    private const ACTION_LIST = 128;
+    private const ACTION_BATCH = 256;
+    private const INTERNAL_ACTIONS = [
+        'tree' => self::ACTION_TREE,
+        // 'create' => self::ACTION_CREATE, // not used now
+        'show' => self::ACTION_SHOW,
+        'edit' => self::ACTION_EDIT,
+        'delete' => self::ACTION_DELETE,
+        'acl' => self::ACTION_ACL,
+        'history' => self::ACTION_HISTORY,
+        'list' => self::ACTION_LIST,
+        'batch' => self::ACTION_BATCH,
+    ];
+    private const ACTION_CREATE_IN = self::ACTION_TREE | self::ACTION_SHOW | self::ACTION_EDIT | self::ACTION_DELETE | self::ACTION_LIST | self::ACTION_BATCH;
```

Comment:
What the `_IN` suffix means in this context?

Reply:
action IN actions

Reply:
IMO, `ACTION_CREATE_MASK` (or `MASK_ACTION_CREATE`) might be a better fit for the purpose of this constant.